src/component/autocomplete.js

```diff
@@ -150,8 +152,8 @@ export class AutoCompleteCustomElement {
   labelWithMatches(result) {
     let label = this.label(result);
 
-    if (!label.replace) {
-      return '';
+    if (typeof label !== 'string') {
+      return ' ';
```

Comment:
So why can't it be an empty string? If it's a workaround, either fix the issue or document why the workaround is needed.